server/routes/overview/mappers/supportNeedsMapper.ts

```diff
@@ -0,0 +1,32 @@
+import type { Neurodiversity, SupportNeeds } from 'viewModels'
+import type { LearnerNeurodivergence, LearnerProfile } from 'curiousApiClient'
+
+const toSupportNeeds = (
+  learnerProfile: LearnerProfile,
+  learnerNeurodivergence: LearnerNeurodivergence,
+): SupportNeeds => {
+  return {
+    languageSupportNeeded: false, // TODO - how do we derive this?
+    lddAndHealthNeeds: toLddAndHealthNeeds(learnerProfile),
+    neurodiversity: toNeurodiversity(learnerNeurodivergence),
+  }
+}
+const toNeurodiversity = (learnerNeurodivergence: LearnerNeurodivergence): Neurodiversity => {
+  if (learnerNeurodivergence) {
+    return {
+      supportNeeded: learnerNeurodivergence.neurodivergenceSupport,
+      selfDeclaredNeurodiversity: learnerNeurodivergence.neurodivergenceSelfDeclared,
+      assessedNeurodiversity: learnerNeurodivergence.neurodivergenceAssessed,
+    }
+  }
+  return undefined
+}
+
+const toLddAndHealthNeeds = (learnerProfile: LearnerProfile): Array<string> => {
+  if (learnerProfile && learnerProfile.primaryLDDAndHealthProblem) {
+    return [learnerProfile.primaryLDDAndHealthProblem, ...learnerProfile.additionalLDDAndHealthProblems.sort()]
```

Comment:
Do we know what property the `sort` operation is on?

Reply:
I don't follow? My intention is that the primary LDD is listed first and the additional ones are ordered alphabetically. I'm basing this on what Mons sent me: https://mojdt.slack.com/archives/C02497NS0HK/p1689662633047629?thread_ts=1689607205.601469&cid=C02497NS0HK